build.gradle

```diff
@@ -316,15 +314,29 @@ subprojects {
     testImplementation 'org.junit.jupiter:junit-jupiter-params'
 
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+
+    integrationTestImplementation 'org.assertj:assertj-core'
+    integrationTestImplementation 'org.apache.tuweni:tuweni-junit'
+    integrationTestImplementation 'org.junit.jupiter:junit-jupiter-api'
+    integrationTestImplementation 'org.junit.jupiter:junit-jupiter-params'
+
+    integrationTestRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
   }
 
-  task integrationTest(type: Test, dependsOn:["compileTestJava"]){
+  task integrationTest(type: Test, dependsOn:["compileIntegrationTestJava"]){
     group = "verification"
     description = "Runs the Artemis integration tests"
 
     testClassesDirs = sourceSets.integrationTest.output.classesDirs
     classpath = sourceSets.integrationTest.runtimeClasspath
-    outputs.upToDateWhen { false }
```

Comment:
Allow the normal upToDate check to function so we don't re-run tests if nothing has changed (matching behaviour for unit tests).